### PR TITLE
cloud: split connection docs

### PR DESCRIPTION
**Pull request by ran-huang**
### What is changed, added or deleted? (Required)

This PR includes the following changes:

- Split the connection docs. Now each connection method has its own document, including:
    - Standard connection
    - Private Endpoint
    - VPC Peering
    - SQL Shell
    
    (SQL editor does not have a separate doc).
- Refine the _connection overview_ doc. It now has two subheadings, one for serverless tier, one for dedicated tier.
- Move some content from the *connection overview* into _Private endpoint_ and _VPC peering_ so that they are more self-contained.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.3 (TiDB 6.3 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/connect-via-sql-shell.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Connect via SQL Shell
+summary:
+---
+
+# Connect via SQL Shell
+
+> **Note:**
+>
+> You cannot connect to [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) using SQL Shell.
```
Comment (Oreoxmt) [inline tidb-cloud/connect-via-sql-shell.md:10]:
```suggestion
> You cannot connect to [Serverless Tier clusters](/tidb-cloud/select-cluster-tier.md#serverless-tier-beta) using SQL Shell. To connect to your Serverless Tier cluster, see [Connect to Serverless Tier clusters](/tidb-cloud/connect-to-tidb-cluster.md#serverless-tier).
```

File: tidb-cloud/set-up-private-endpoint-connections.md [hunk 2]
```diff
@@ -164,7 +168,19 @@ aws ec2 modify-vpc-endpoint --vpc-endpoint-id <your_vpc_endpoint_id> --private-d
 
 Click **Create** in the TiDB Cloud console to finalize the creation of the private endpoint.
 
-Then you can connect to the endpoint service. See [Connect to TiDB cluster via private endpoint](/tidb-cloud/connect-to-tidb-cluster.md#connect-via-private-endpoint-recommended) for details.
+Then you can connect to the endpoint service.
+
+### Step 6: Connect to your TiDB cluster
```
Comment (TomShawn) [inline tidb-cloud/set-up-private-endpoint-connections.md:173]: Add this title to Line 48 of this file?

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 3]
```diff
@@ -194,13 +200,9 @@ You can also use the AWS dashboard to configure the VPC peering connection.
 
     5. From the setting drop-down list, click **Edit DNS resolution**. Enable DNS resolution and click **Save**.
 
-### Step 3. Connect to the TiDB cluster on TiDB Cloud
-
-1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page of your target project.
-2. Locate the target cluster, and click **Connect** in the upper-right corner of the cluster area. The connection dialog is displayed. You can see the **Status** of the VPC peering is **active**. If **Status** is still **system checking**, wait for about 5 minutes and open the dialog again.
-3. Access the TiDB Cluster from the instance within the VPC. See [Connect to Your TiDB Cluster](/tidb-cloud/connect-to-tidb-cluster.md).
+Now you have successfully set up the VPC peering connection. Next, [connect to the TiDB cluster  via VPC peering](#connect-to-the-tidb-cluster).
```
Comment (TomShawn) [inline tidb-cloud/set-up-vpc-peering-connections.md:203]:
```suggestion
Now you have successfully set up the VPC peering connection. Next, [connect to the TiDB cluster via VPC peering](#connect-to-the-tidb-cluster).
```

File: tidb-cloud/set-up-private-endpoint-connections.md (hunk 2 above)
Comment (ran-huang) [reply tidb-cloud/set-up-private-endpoint-connections.md:174]: done

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 4]
```diff
@@ -1,135 +1,37 @@
...
+    Serverless Tier only [supports TLS connections](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md), which ensures the security of data transmission from your applications to TiDB clusters.
 
-    > **Note:**
-    >
-    > - The TiDB cluster CA is only available for Dedicated Tier clusters.
-    > - Currently, TiDB Cloud only provides the connection strings and sample code for these connection methods: MySQL, MyCLI, JDBC, Python, Go, and Node.js.
+- Connect via SQL editor
 
-5. Under **Step 3: Connect with a SQL client** in the dialog, click the tab of your preferred connection method, and then refer to the connection string and sample code on the tab to connect to your cluster.
+    SQL editor is a web-based SQL editor that allows you to directly edit and run SQL queries against databases of Serverless Tier. To access SQL editor, in the [TiDB Cloud console](https://tidbcloud.com/), navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page, locate your cluster, click **Connect** and then select **SQL Editor** in the drop-down list.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:23]: Btw, although accessing SQL editor is simple, it is still worth creating a separate doc to promote it to users, just as "Connect via SQL Shell".

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 5]
```diff
@@ -1,135 +1,37 @@
...
 
-    Note that you need to use the path of the downloaded CA file as the argument of the `--ssl-ca` option in the connection string.
+## Dedicated Tier
 
-</div>
-</SimpleTab>
+For Dedicated Tier clusters, you can connect to your cluster via a SQL client or via SQL Shell in the TiDB Cloud console.
```
[the remark this answers is not in the document]
Comment (TomShawn) [reply tidb-cloud/connect-to-tidb-cluster.md:26]: +1

**Review by TomShawn — APPROVED** [review]
LGTM for the private endpoint update

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 6]
```diff
@@ -1,135 +1,37 @@
...
+    The standard connection exposes a public endpoint with traffic filters, so you can connect to your TiDB cluster from your laptop.
 
-4. Under **Step 2: Download TiDB cluster CA** in the dialog, click **Download TiDB cluster CA** for TLS connection to TiDB clusters. The TiDB cluster CA supports TLS 1.2 version by default.
+    Serverless Tier only [supports TLS connections](/tidb-cloud/secure-connections-to-serverless-tier-clusters.md), which ensures the security of data transmission from your applications to TiDB clusters.
 
-    > **Note:**
-    >
-    > - The TiDB cluster CA is only available for Dedicated Tier clusters.
-    > - Currently, TiDB Cloud only provides the connection strings and sample code for these connection methods: MySQL, MyCLI, JDBC, Python, Go, and Node.js.
+- Connect via SQL editor
```
Comment (ran-huang) [inline tidb-cloud/connect-to-tidb-cluster.md:20]:
```suggestion
- Connect via SQL editor (Beta)
```

File: tidb-cloud/connect-to-tidb-cluster.md (hunk 4 above)
Comment (ran-huang) [reply tidb-cloud/connect-to-tidb-cluster.md:23]: Maybe we can wait until the SQL editor is GA? There might be drastic changes to the UI and features before then.

File: tidb-cloud/set-up-vpc-peering-connections.md [hunk 7]
```diff
@@ -1,17 +1,21 @@
 ---
-title: Set Up VPC Peering Connections
-summary: Learn how to set up VPC peering connections.
+title: Connect via VPC Peering Connections
+summary: Learn how to connect to TiDB Cloud via VPC peering connections.
 ---
 
-# Set Up VPC Peering Connections
+# Connect via VPC Peering Connections
```
Comment (ran-huang) [inline tidb-cloud/set-up-vpc-peering-connections.md:6]:
```suggestion
# Connect via VPC Peering
```

File: tidb-cloud/connect-to-tidb-cluster.md (hunk 4 above)
Comment (qiancai) [reply tidb-cloud/connect-to-tidb-cluster.md:23]: +1

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 8]
```diff
@@ -1,135 +1,36 @@
...
 
-+ Connect via a SQL client
+## Serverless Tier
 
-    - [Connect via standard connection](#connect-via-standard-connection): The standard connection exposes a public endpoint with traffic filters, so you can connect to your TiDB cluster from your laptop. You can connect to your TiDB clusters using TLS, which ensures the security of data transmission from your applications to TiDB clusters.
-    - [Connect via private endpoint](#connect-via-private-endpoint-recommended) (only available for Dedicated Tier): Private endpoint connection provides a private endpoint to allow clients in your VPC to securely access services over AWS PrivateLink, which provides highly secure and one-way access to database services with simplified network management.
-    - [Connect via VPC peering](#connect-via-vpc-peering) (only available for Dedicated Tier): If you want lower latency and more security, set up VPC peering and connect via a private endpoint using a VM instance on the corresponding cloud provider in your cloud account.
+For Serverless Tier clusters, you can connect to your cluster via standard connection or via SQL editor in the TiDB Cloud console.
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:12]:
```suggestion
For Serverless Tier clusters, you can connect to your cluster via standard connection or via SQL editor (Beta) in the TiDB Cloud console.
```

File: tidb-cloud/connect-to-tidb-cluster.md [hunk 9]
```diff
@@ -1,135 +1,36 @@
...
-To connect to your TiDB cluster using SQL Shell, perform the following steps:
-
-1. Navigate to the [**Clusters**](https://tidbcloud.com/console/clusters) page and choose your target project on the top of the left navigation bar.
-
-2. Locate your cluster, click **Connect** in the upper-right corner of the cluster area, and select the **Web SQL Shell** tab in the connection dialog.
-
-3. Click **Open SQL Shell**.
-
-4. On the prompted **TiDB password** line, enter the root password of the current cluster. Then your application is connected to the TiDB cluster.
++ [Connect via SQL Shell](/tidb-cloud/connect-via-sql-shell.md): to try TiDB SQL and test out TiDB's compatibility with MySQL quickly, or administer user privileges
```
Comment (qiancai) [inline tidb-cloud/connect-to-tidb-cluster.md:33]:
```suggestion
+ [Connect via SQL Shell](/tidb-cloud/connect-via-sql-shell.md): to try TiDB SQL and test out TiDB's compatibility with MySQL quickly, or administer user privileges.
```

File: tidb-cloud/connect-via-sql-shell.md [hunk 10]
```diff
@@ -0,0 +1,20 @@
+---
+title: Connect via SQL Shell
+summary: Learn how to connect to your TiDB cluster via SQL Shell.
+---
+
+# Connect via SQL Shell
+
```
Comment (qiancai) [inline tidb-cloud/connect-via-sql-shell.md:7]:
```suggestion

In TiDB Cloud SQL Shell, you can try TiDB SQL, test out TiDB's compatibility with MySQL quickly, and administer database user privileges.
```

**Review by qiancai — APPROVED** [review]
other LGTM

File: tidb-cloud/connect-via-standard-connection.md [hunk 11]
```diff
@@ -0,0 +1,64 @@
+---
+title: Connect via Standard Connection
+summary: Learn how to connect to your TiDB Cloud cluster via standard connection.
+---
+
+# Connect via Standard Connection
+
+This document describes how to connect to your TiDB Cloud cluster via standard connection. The standard connection exposes a public endpoint with traffic filters, so you can connect to your TiDB cluster via a SQL client from your laptop.
+
+Standard connection is available to both Serverless Tier and Dedicated Tier.
```
Comment (qiancai) [inline tidb-cloud/connect-via-standard-connection.md:10]:
```suggestion
The standard connection is available to both Serverless Tier and Dedicated Tier.
```

File: tidb-cloud/connect-via-sql-shell.md [hunk 12]
```diff
@@ -0,0 +1,21 @@
...
+In TiDB Cloud SQL Shell, you can try TiDB SQL, test out TiDB's compatibility with MySQL quickly, and administer database user privileges.
```
Comment (Oreoxmt) [inline tidb-cloud/connect-via-sql-shell.md:8]:
```suggestion
In TiDB Cloud SQL Shell, you can try TiDB SQL, test out TiDB's compatibility with MySQL quickly, and administer database user privileges.

```

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Pull request merged**